src/pages/home/report/ReportActionsView.js

```diff
@@ -252,30 +251,6 @@ class ReportActionsView extends React.Component {
         Report.unsubscribeFromReportChannel(this.props.report.reportID);
     }
 
-    /**
-     * @param {Object} reportActions
-     * @returns {Array}
-     */
-    getSortedReportActionsForDisplay(reportActions) {
-        // HACK ALERT: We're temporarily filtering out any reportActions keyed by sequenceNumber
-        // to prevent bugs during the migration from sequenceNumber -> reportActionID
-        const filteredReportActions = _.filter(reportActions, (reportAction, key) => {
-            if (!reportAction) {
-                return false;
-            }
-
-            if (String(reportAction.sequenceNumber) === key) {
-                Log.info('Front-end filtered out reportAction keyed by sequenceNumber!', false, reportAction);
```

Comment:
@tienifr Any reason why we delete the log? It's just got updated 2 days ago in this PR https://github.com/Expensify/App/pull/15146

Reply:
@mollfpr Because this log is a new update, I don't see that before. I will add it again